### DISCUSSION TITLE
How could I upload all the cities for a dropdown field?

### DISCUSSION THREAD

**Discussion by OscarGS86** [YetiForceCompany/YetiForceCRM#15559] [How to]
Hi Guys,

I would like to know the best way to upload all the cities to be used in a dropdown field for the Contacts section to attach each contact with the correct city.

Do we have any way to do any kind of integration to do it automatically? or we need to do create a field manually adding all the cities?

Thanks for your help

**Comment by vovpff** [comment]
You can do it using sql

**Comment by mariuszkrzaczkowski** [comment]
New city module + reference field

**Reply by OscarGS86** [reply]
Hi @mariuszkrzaczkowski , 

Please, could you send me more info to know how to do it?

Thank you

**Reply by mariuszkrzaczkowski** [reply]
Create a new module named "cities" and add a reference field in your module where you want to be linked to the city module
